flowmachine/flowmachine/features/subscriber/handset_stats.py

```diff
@@ -4,7 +4,7 @@
 
 # -*- coding: utf-8 -*-
 
-valid_stats = {"count", "sum", "avg", "max", "min", "median", "stddev", "variance"}
+valid_stats = {"count", "sum", "avg", "max", "min", "stddev", "variance"}
```

Comment:
Would be nice to support weighted median here, I suppose. But that's out of scope for this PR, and quite possibly something that would never actually be used.

Reply:
Could do with updating the docstring accordingly (I'd add a 'suggestion' comment if GitHub would let me)

Reply:
Hang on, no, I think median _is_ supported here. `_make_query` is written in an exceedingly convoluted way, but I think the final `return sql` (only reached if all the `if self.statistic in ...` are `False`) is the weighted median. The same is true in `TopupBalance`.

Docstring could still do with an update, though - it lists 'mode', which is not supported here (although there's no reason it couldn't be; it's implemented in `TopupBalance`).

Reply:
Oh good lord you're right. Well spotted. I'll reinstate and add a comment to make that more obvious I think.